geode-core/src/main/java/org/apache/geode/internal/net/SCClientSocketCreator.java

```diff
@@ -12,16 +12,24 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.distributed.internal.membership.gms.membership;
+package org.apache.geode.internal.net;
 
-import java.net.InetSocketAddress;
+import java.io.IOException;
+import java.net.Socket;
 
-import org.apache.geode.distributed.internal.tcpserver.LocatorAddress;
+import org.apache.geode.distributed.internal.tcpserver.ClientSocketCreatorImpl;
 
-public class HostAddress extends LocatorAddress {
+class SCClientSocketCreator extends ClientSocketCreatorImpl {
```

Comment:
SocketCreator.  I wanted to rename all of this since the purpose of SocketCreator is to add TLS support but it would kill backward-compatibility tests to change SocketCreator's name.  Maybe we can do that in Geode 2.0.